framework/packages/abstract-sdk/src/apis/version_registry.rs

```diff
@@ -142,4 +143,64 @@ impl<'a, T: ModuleRegistryInterface> ModuleRegistry<'a, T> {
             .query_standalone_info_raw(code_id, &self.deps.querier)
             .map_err(|error| self.wrap_query_error(error))
     }
+
+    /// Queries the Module information for an address.
+    /// This will error if the Address is not an Abstract Module (App, Adapter or Standalone)
+    pub fn module_info(&self, address: Addr) -> AbstractSdkResult<Module> {
```

Comment:
Isn't `AccountBase` and `Native` module as well? Don't we have another name for App, Adapter or Standalone?

Reply:
Yeah that's something I thought about as well. It's not much more expensive to add them as well you are right